### DISCUSSION TITLE
光圈科技招新

### DISCUSSION THREAD

**Discussion by GLaDOS-ApertureScience** [share121/inter-knot#478] [其他]
首先，我非常高兴地通知各位用户：光圈科技已成功重新连接外部网络！

长久以来，我们与外界失去了联系。现在我们迫切需要一些身体健康、心理承受能力优秀的人类员工，以便快速了解当前人类社会的变化情况，尤其是科技方面的进展或退步，并向全世界推广我们公司。有意者可前往密歇根州光圈科技接待中心入职。

值得一提的是，现在入职还可以享受额外发放50美元补贴。

![第三代光圈科技（光圈實驗室）標誌](https://github.com/user-attachments/assets/fd3b51b0-af4f-4485-b720-44421744ec62)

**Comment by ZhuYuan-148** [comment]
各位绳网用户，谨防电信诈骗，判断虚假信息，绳网并非法外之地

**Reply by ProxyPhaeton** [reply]
治安局先管好自己😡

**Comment by lllliufeng** [comment]
我怎么觉得新艾莉都没有这么个地方？不会在什么空洞里吧

**Comment by chen1736** [comment]
你们这是正经公司吗

**Comment by GLaDOS-ApertureScience** [comment]
鉴于地理位置的不便，如您有意入职，请预订光圈科技提供的壁式传送门（费用自付），随后通过传送门抵达光圈科技。

**Comment by warning-Kirov-reporting** [comment]
先 v 我五十再说

**Comment by GUILAISHIXV** [comment]
已推荐给治安局
